### ISSUE TITLE
NHL completely dead at this point?

### ISSUE THREAD

**Issue by mlindgren80** [tarkah/lazystream#113]
Since the start of the season I haven’t been able to watch any streams at all. It either says stream not yet available or “couldn’t find any”

is this method of viewing streams dead at this point?

using .12.1 on macOS and Linux.

**Comment by 13Hemi** [comment]
Same issue here. Also noticed that when you browse manually to the stream URL in a web browser the site redirects to some phishing ad sites. Looks like it may have gotten hijacked.

**Comment by un999** [comment]
@mlindgren80 yes the source provider is no longer responding on reddit and it is presumed abandoned.  There is another source the 66 one that a clever person was able to parse out the direct links that can be played directly with vlc or mpv.  So far only available live and requires vpn that is not US or CAN.  They are working on replays.

**Comment by eblanchette** [comment]
@un999 "the 66 one"?  do you have a link?

**Comment by un999** [comment]
The source provider is reddit dot com / r / nhl underscore games, there is a wealth of information about this issue and many alternatives in the posts starting around 2022-09-27

The 66 site is nhl 66 dot ir and the clever person made a site nhl 66 dot ga that parses the links from that source.

**Comment by bob1321** [comment]
Can someone make something work with all these link? nhl 66 dot ga

**Comment by CrashBandicooch** [comment]
> Can someone make something work with all these link? xyz.ab

I believe people are working on that. But you should delete the URL from your comment to ensure it isn't indexed by any search engines, etc.

Read through this issue thread from the beginning and you'll see resources to follow developments.

**Comment by mlindgren80** [comment]
Interesting, I'm VPNd to Sweden and nhl 66 dot ga links are still showing up as permission denied.

**Comment by bob1321** [comment]
> Interesting, I'm VPNd to Sweden and nhl 66 dot ga links are still showing up as permission denied.

https://www.nhl.com/info/where-to-stream
Sweden dont have access to NHL.TV
Try UK VPN

**Comment by bob1321** [comment]
> > > Interesting, I'm VPNd to Sweden and nhl 66 dot ga links are still showing up as permission denied.
> > 
> > 
> > https://www.nhl.com/info/where-to-stream Sweden dont have access to NHL.TV Try UK VPN
> 
> I can access the links from a country that's not on the "available" list on that page. So looks like the geo blacklist is a different set of countries.

The only problem is we need to refresh the link often because the key is changing
